recipes/tiny-optional/all/conanfile.py

```diff
@@ -0,0 +1,40 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.layout import basic_layout
+from conan.tools.files import copy, get
+import os
+
+
+required_conan_version = ">=2.0.9"
+
+
+class TinyOptionalConan(ConanFile):
+    name = "tiny-optional"
+    description = "Replacement for std::optional that does not waste memory unnecessarily"
+    license = "BSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Sedeniono/tiny-optional"
+    topics = ("optional", "header-only")
```

Comment:
```suggestion
    topics = ("optional", "memory-efficiency", "cache-friendly", "header-only")
```
Can include these topics from their GitHub tags.